sdk/core/core/test/test_mut_span.c

```diff
@@ -0,0 +1,31 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
again I want to see more tests for the failures, and failures followed by more usages of the span builder.